tst/functions/Mock.Tests.ps1

```diff
@@ -899,6 +899,18 @@ Describe "When Calling Should -Not -Invoke -ExclusiveFilter" {
     }
 }
 
+Describe "When Calling Should -Invoke with pipeline-input or -ActualValue" {
+    It "Should throw an error on pipeline-input" {
+        $scriptBlock = { "FunctionUnderTest" | Should -Invoke -CommandName "ABC" -Scope Describe }
```

Comment:
```suggestion
        $scriptBlock = { "value" | Should -Invoke -CommandName "ABC" -Scope Describe }
```

I get where you got the "FunctionUnderTest" from, but it is confusing without having that context.

Reply:
Agreed. Had the mock there at first, but removed it to avoid unnecessary processing.